07-bootstrap-validation.md

```diff
@@ -22,8 +22,8 @@ GitOps allows a team to author Kubernetes manifest files, persist them in their
 1. Get the cluster name.
 
    ```bash
-   AKS_CLUSTER_NAME=$(az aks list -g rg-bu0001a0008 --query '[0].name' -o tsv)
-   echo AKS_CLUSTER_NAME: $AKS_CLUSTER_NAME
+   export AKS_CLUSTER_NAME_AKS_BASELINE=$(az aks list -g rg-bu0001a0008 --query '[0].name' -o tsv)
```

Comment:
Why are we exporting now?

Reply:
because https://github.com/mspnp/aks-baseline/blob/main/saveenv.sh save the variables that are exported and end by _AKS_BASELINE. 
Without the export, the variables are not saved. 
That is good when you for some reason need to re-start a deploy process in the middle, you have the variable values till the moment you executed the latest  saveenv.sh.